docs/QuickReference.html

```diff
@@ -2,92 +2,216 @@
+<p>Operations on sets include + (union), * (intersection), and &#8211; (set difference), as well as the set comparison operators < (proper subset), <= (subset), their duals > and >=, and !! (disjointness). 
+The expression <code>x in S</code> says that x is a member of set S, and <code>x !in S</code> is a convenient way of writing <code>!(x in S)</code>. 
+To make a set from some elements, enclose them in curly braces. 
+For example, <code>{x,y}</code> is the set consisting of x and y (which is a singleton set if <code>x == y</code>), <code>{x}</code> is the singleton set containing x, and <code>{}</code> is the empty set.</p>
+
+<p>Operations on sequences include + (concatenation) and the comparison operators < (proper prefix) and <= (prefix). 
+Membership can be checked like for sets: <code>x in S</code> and <code>x !in S</code>. 
+The length of a sequence S is denoted <code>|S|</code>, and the elements of such a sequence have indices from 0 to less than <code>|S|</code>. 
+The expression <code>S[j]</code> denotes the element at index j of sequence S. 
+The expression <code>S[m..n]</code>, where 0 <= m <= n <= |S|, returns a sequence whose elements are the n-m elements of S starting at index m (that is, from S&#091;m&#093;, S&#091;m+1&#093;, ... up to but not including S&#091;n&#093;). 
+The expression S&#091;m..&#093; (often called "drop m") is the same as S&#091;m..|S|&#093;, that is, it returns the sequence whose elements are all but the first m elements of S. 
+The expression S&#091;..n&#093; (often called "take n") is the same as S&#091;0..n&#093;, that is, it returns the sequence that consists of the first n elements of S. 
```

Comment:
These should be formatted like in the previous lines

Reply:
Done